future_goals.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+   <title>Alan Turing - Past and Future</title>
+   <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+  
+<body>
+  <nav>
+    <a href="index.html">Past Achievements</a>
+    *
+    <strong>Future Goals</strong> 
+  </nav>
+
+  <h1>Alan <strong>M.F.</strong> Turing</h1>
+  
+  <hr>
+
+  <h2>Future Goals</h2>
+
+  <p>
+    This part is complicated. I'm currently dead, but if I was to return to life, I'd do at least these three things (in no particular order):
+      <ol>
+        <li>I'd work with leading researchers to further my research on AI.</li>
+        <li>I'd study the current state of encryption technology and develop enhanced security for all electronic communications.</li>
+        <li>I'd love to try a burrito -- we didn't really have the best Mexican food in the UK in the 1940s and 50s.</li>
+      </ol>
+    <img src="images/burritos.jpg"/>
```

Comment:
Although not required, the best practice is to include the `alt` attribute which allows for accessibility to the visually impaired and is what appears if the resource fails to load. 🔨